web/src/routes/(user)/albums/+page.svelte

```diff
@@ -114,17 +132,53 @@
       </div>
     </LinkButton>
 
-    <Dropdown options={sortByOptions} bind:value={$albumViewSettings.sortBy} />
+    <Dropdown options={sortByOptions} bind:value={$albumViewSettings.sortBy} icons={[SwapVertical]} />
```

Comment:
Do you think there would be any other views besides List and Cover? if not, I'd rather click on the button to switch between the two. This brings 2 benefits

1. Less code to maintain
2. Less click to achieve a goal

Reply:
Actually, this is fine, you can disregard the comment